frontend/components/page-contents/feedback-task-content/RecordFeedbackTaskAndQuestionnaire.content.vue

```diff
@@ -86,6 +92,7 @@ export default {
   data() {
     return {
       rerenderQuestionnaire: 1,
+      transitionWhenNavigating: true,
```

Comment:
1/ renaming : 'isTransition' (the component does not know that the transition is used on pagination,or other way)
2/ can you pass it as a props with a default value as true ? because when we will want to remove the transition, the logic for it need to come from the parent